### PR TITLE
ToolCategory Model

### PR DESCRIPTION
**Pull request by connordibble**
I made a basic model and configured django with the usual django stuff.  I also made a template file index.html that will work with the init view /tools/init.  Run it and make sure it's working for you all.  Thanks!

### CODE REVIEW

File: toolsWebsite/tools/models.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
-# Create your models here.
+class ToolCategory(models.Model):
+    type = models.CharField(max_length=30)
+    count = models.IntegerField(default=0)
+    unavailable = models.IntegerField(default=0)
```
Comment (tgreen24) [inline toolsWebsite/tools/models.py]: We should add a price field here as well, so we can say how much each tool will cost.

File: toolsWebsite/tools/templates/tools/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <link rel="stylesheet" href="{% static 'tools/styles.css' %}" type="text/css"/>
+
+</head>
+
+<body>
+  <h1>List of Tools</h1>
+{% for category in tools_list %}
+  <br>
+  <h4 style="font-size: large">{{ category.type }}</h4>
+  <p style="font-size: large">Available: {{ category.count }}</p>
```
Comment (tgreen24) [inline toolsWebsite/tools/templates/tools/index.html]: Wouldn't the number of available tools be `category.count - category.unavailable`?

File: toolsWebsite/tools/models.py (hunk 1 above)
Comment (connordibble) [reply toolsWebsite/tools/models.py]: Finished that up.

File: toolsWebsite/tools/templates/tools/index.html (hunk 2 above)
Comment (connordibble) [reply toolsWebsite/tools/templates/tools/index.html]: Count was a bad variable name.  Count is actually the number of tools available.  (i.e. there could be 3 available and 5 checked out)

**Pull request merged**